libs/watt/package/dropdown/watt-dropdown.component.ts

```diff
@@ -167,15 +139,14 @@ export class WattDropdownComponent implements ControlValueAccessor, OnInit {
    * Set the mode of the dropdown.
    */
   chipMode = input(false);
-
   disableSelectedMode = input(false);
-
   sortDirection = input<'asc' | 'desc'>();
 
   /**
    * Sets the options for the dropdown.
+   * Can be a flat array of options or an array containing both options and option groups.
    */
-  options = model<WattDropdownOptions>([]);
+  options = model<WattDropdownGroupedOptions>([]);
```

Comment:
question: should it not be `model<WattDropdownOptions | WattDropdownGroupedOptions>`?